dts/msm8916-samsung-r06.dts

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "msm8916.dtsi"
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <206 0>;
+	qcom,board-id = <0xCE08FF01 6>;
+
+	gt58ltebmc {
+		model = "Samsung Galaxy Tab A 8.0";
```

Comment:
Indicating that this is the Canadian variant might be helpul. So either something like 

```suggestion
		model = "Samsung Galaxy Tab A 8.0 (CAN)";
```

or 

```suggestion
		model = "Samsung Galaxy Tab A 8.0 (SM-T357W)";
```

Just make sure that it fits on one line in the fastboot screen.